ultrasonic/src/main/kotlin/org/moire/ultrasonic/service/MediaPlayerService.kt

```diff
@@ -95,6 +96,12 @@ class MediaPlayerService : Service() {
 
         localMediaPlayer.onNextSongRequested = Runnable { setNextPlaying() }
 
+        // TODO maybe MediaSession must be in an independent class after all...
```

Comment:
I think one of the next steps could be to refactor the MediaSession to its own handler class if that's possible...